ysoserial/Helpers/YamlDotNet.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Text.RegularExpressions;
+
+// Coded by Soroush Dalili (@irsdl)
+namespace ysoserial.Helpers
+{
+    class YamlDotNet
+    {
+        public static String Minify(String xmlDocument)
```

Comment:
rename to `yamlDocument`?

Reply:
I was not sure whether this minimisation is generic or just related to .NET tbh. I will rename it on the next PR if this is what you want (just let me know please).

Reply:
yep, just for clarity, since it has nothing to do with `xml`. You dont need a new PR, just add a new commit to your fork and the PR will get updated